src/components/biblio/topic_entity_tag/TopicEntityTable.js

```diff
@@ -35,10 +35,9 @@ const TopicEntityTable = () => {
   const [selectedSpecies, setSelectedSpecies] = useState([]);
   const [speciesFilterPosition, setSpeciesFilterPosition] = useState({ top: 0, left: 0 });
   const [allSpecies, setAllSpecies] = useState([]);
-  const defaultPageSize = 25;
-  //const [pageSize, setPageSize] = useState(defaultPageSize);
-  const pageSize = storedPageSize || defaultPageSize;
-  
+  //const defaultPageSize = 25;
+  //const pageSize = storedPageSize || defaultPageSize;
+  const pageSize = storedPageSize;
```

Comment:
pageSize could just be directly set in line 27 from useSelector, but this is all great, thanks @sweng66 !

Reply:
you are right! lol @azurebrd

Reply:
@azurebrd Fixed. Sorry. back/forth so many times!